src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -330,4 +330,23 @@ function rhs_parabolic!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabol
 
     return nothing
 end
+
+function _jacobian_ad_forward(semi::SemidiscretizationHyperbolicParabolic, t0, u0_ode,
+                              du_ode, config)
+    new_semi = remake(semi, uEltype = eltype(config))
+
+    du_ode_hyp = Vector{eltype(config)}(undef, length(du_ode))
+    J = ForwardDiff.jacobian(du_ode, u0_ode, config) do du_ode, u_ode
+        for i in eachindex(du_ode)
+            du_ode_hyp[i] = du_ode[i]
+        end
+
```

Comment:
```suggestion
```
I don't see a reason to do this since you overwrite it later?

Reply:
Probably unnecessary, was not sure due to the AD datatypes.